proto/v1/database_edit.proto

```diff
@@ -0,0 +1,206 @@
+syntax = "proto3";
+
+package bytebase.v1;
+
+import "google/api/field_behavior.proto";
+
+option go_package = "generated-go/v1";
+
+message DatabaseEdit {
+  // The name of the database to retrieve.
+  // Format: instances/{instance}/databases/{database}.
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
This edits don't need a name any more since we only need context.

Reply:
removed